launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml

```diff
@@ -296,7 +296,8 @@
       <arg name="input/image6" value="$(var image_raw6)"/>
       <arg name="input/image7" value="$(var image_raw7)"/>
       <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
-      <arg name="output/objects" value="objects"/>
```

Comment:
[Q] It works fine, but is there any reason to remove this line?

Reply:
Sorry it was a complete mistake. Fixed: https://github.com/autowarefoundation/autoware.universe/pull/6106/commits/22f4c0d3180363c06700d7dd45641c3ebb2d405a